console/src/components/Services/Settings/Actions.ts

```diff
@@ -50,41 +54,48 @@ const reloadRemoteSchemaCacheAndGetInconsistentObjectsQuery = remoteSchemaName =
   };
 };
 
-const getReloadCacheAndGetInconsistentObjectsQuery = shouldReloadRemoteSchemas => ({
+const getReloadCacheAndGetInconsistentObjectsQuery = (
+  shouldReloadRemoteSchemas: boolean
+) => ({
   type: 'bulk',
   args: [
     getReloadMetadataQuery(shouldReloadRemoteSchemas),
     inconsistentObjectsQuery,
   ],
 });
 
-export const exportMetadata = (successCb, errorCb) => (dispatch, getState) => {
+export const exportMetadata = (successCb: Function, errorCb: Function) => (
+  dispatch: ThunkDispatch<any, any, AnyAction>,
+  getState: () => AppState
+) => {
   const { dataHeaders } = getState().tables;
 
-  const query = exportMetadataQuery;
-
-  const options = {
+  const options: RequestInit = {
     method: 'POST',
     headers: {
       ...dataHeaders,
     },
-    body: JSON.stringify(query),
+    body: JSON.stringify(exportMetadataQuery),
   };
 
   dispatch(requestAction(endpoints.query, options))
     .then(response => {
       successCb(response);
     })
-    .catch(err => {
+    .catch((err: Error) => {
       errorCb(err);
     });
 };
 
-export const replaceMetadata = (newMetadata, successCb, errorCb) => (
-  dispatch,
-  getState
+export const replaceMetadata = (
+  newMetadata: object,
+  successCb: Function,
```

Comment:
Same here, with this difference that both `successCb` and `errorCb` can be typed as `() => void`.

```ts
export const replaceMetadata = (
  newMetadata: object,
  successCb: () => void,
  errorCb: () => void
) => (
  dispatch: ThunkDispatch<ReduxState, {}, AnyAction>,
  getState: GetReduxState
)
```